npm/packs/jquery/src/abp.jquery.js

```diff
@@ -106,7 +106,7 @@ var abp = abp || {};
                     $dfd.resolve(data);
                     userOptions.success && userOptions.success(data);
                 }).fail(function (jqXHR) {
-                    if(jqXHR.status === 0 || jqXHR.statusText === 'abort') {
+                    if(jqXHR.statusText === 'abort') {
```

Comment:
We can consider adding an `error callback` to allow developers to handle it themselves.

eg:
`userOptions.success && userOptions.success(data);`